src/helpers/tokenUtils.ts

```diff
@@ -0,0 +1,294 @@
+import useWeb3 from '@/compositions/useWeb3'
+import useErc725 from '@/compositions/useErc725'
+import { eip165ABI } from '@/abis/eip165ABI'
+import { erc20ABI } from '@/abis/erc20ABI'
+import BN from 'bn.js'
+import { store, setState } from '../stores/index'
+import { ERC725JSONSchema } from '@erc725/erc725.js'
+import {
```

Comment:
is it possible to order the imports with:

```
external dep

internal dep
```

?
I find it cleaner when we start to have a lot of imports. We can also enforce this in prettier with plugin import

Reply:
We can add the import order thing, but ok I can just change the order here. Adding the eslint thing is a rather large effort for this app.

Reply:
Changed